build-tools/packages/build-cli/docs/transform.md

```diff
@@ -0,0 +1,35 @@
+`flub transform`
+================
+
+Transform commands are used to transform code, docs, etc. into alternative forms.
+
+* [`flub transform releaseNotes`](#flub-transform-releasenotes)
+
+## `flub transform releaseNotes`
+
+Transforms a markdown release notes file into a format appropriate for use in a GitHub Release. This is used to transform in-repo release notes such that they can be automatically posted to our GitHub Releases.
+
+```
+USAGE
+  $ flub transform releaseNotes --inFile <value> --outFile <value> [--json] [-v | --quiet]
+
+FLAGS
+  --inFile=<value>   (required) A release notes file that was generated using 'flub generate releaseNotes'. The file
+                     must not already contain heading links. That is, the input file must not have been generated with
+                     the --headingLinks flag.
+  --outFile=<value>  (required) Output the transformed content to this file.
+
+LOGGING FLAGS
+  -v, --verbose  Enable verbose logging.
+      --quiet    Disable all logging.
+
+GLOBAL FLAGS
+  --json  Format output as json.
```

Comment:
Tangent: I think this refers to the _logging_ output? Might be worth clarifying that if possible (does oclif let us override this flag's docs?), so no one thinks that this could affect the contents of the generated file (same in any other command that produces some output file).

Reply:
Unfortunately it's not easy to override the description of the flag AFAIK. But we probably don't need the flag in this command. The output would be something like `{ "contents": "<Full markdown string of the document>" }`

Reply:
Removed the json flag.